### PR TITLE
Feat(design-tokens): Use CSS variables in themes directly

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by curdaj — APPROVED** [review]

**Comment by literat** [comment]
Are we now able to change the colors from a browser or externally by just setting the variable? Or what is the purpose? There is no description nor linked issue. 🤷 

Also, end-to-end tests are failing.

**Comment by crishpeen** [comment]
> Are we now able to change the colors from a browser or externally by just setting the variable? Or what is the purpose? There is no description nor linked issue. 🤷
> 
> Also, end-to-end tests are failing.

the usage stays the same. This just moves the CSS variables definition from web package to design-tokens package.

Before this PR the design-tokens package expected that the CSS variables are in some format defined by web package. Now all the CSS var definitions are in one place.

I will fix the e2e tests.

File: packages/design-tokens/src/scss/@themes.scss [hunk 1]
```diff
@@ -1,12 +1,15 @@
-@use 'themes/theme-light-default';
-@use 'themes/theme-light-on-brand';
+@use 'sass:meta';
+@use 'themes/theme-light-default' as theme-light-default;
+@use 'themes/theme-light-on-brand' as theme-light-on-brand;
```
Comment (adamkudrna) [inline packages/design-tokens/src/scss/@themes.scss:3]: Is renaming necessary?

**Review by adamkudrna — APPROVED** [review]
Very nice beautiful! 👏🏻

**Comment by literat** [comment]
> the usage stays the same. This just moves the CSS variables definition from web package to design-tokens package.
> 
> Before this PR the design-tokens package expected that the CSS variables are in some format defined by web package. Now all the CSS var definitions are in one place.
> 
> I will fix the e2e tests.

Ok, thanks for the explanation. It will be also nice to link this with the PR where the exporter also solves this. Since this is a manual change the design tokens are generated. Maybe a properly filled description of the PR will help us who were missing for a week or more ;-)

Anyway, thanks for the clarification and fix of the end-to-end test. :-)

**Review by literat — APPROVED** [review]

**Pull request merged**